backend/eventloop/application_event_loop/application_event_runner_test.go

```diff
@@ -72,9 +165,219 @@ var _ = Describe("ApplicationEventLoop Test", func() {
 				testOnlySkipCreateOperation: true,
 			}
 
-			// ------
+			// ----------------------------------------------------------------------------
+			// This should update the existing application
+			// ----------------------------------------------------------------------------
+
+			_, _, _, message, err = appEventLoopRunnerActionSecond.applicationEventRunner_handleDeploymentModified(ctx, dbQueries)
+			Expect(err).To(BeNil())
+			Expect(message).To(Equal("updatedApp"))
+
+			// ----------------------------------------------------------------------------
+			// Verify that the database entries have been updated
+			// ----------------------------------------------------------------------------
+
+			var appMappingsSecond []db.DeploymentToApplicationMapping
+			err = dbQueries.ListDeploymentToApplicationMappingByNamespaceAndName(context.Background(), gitopsDepl.Name, gitopsDepl.Namespace, workspaceID, &appMappingsSecond)
+
+			Expect(err).To(BeNil())
+			Expect(len(appMappingsSecond)).To(Equal(1))
+
+			deplToAppMappingSecond := appMappingsSecond[0]
+			applicationSecond := db.Application{Application_id: deplToAppMappingSecond.Application_id}
+			err = dbQueries.GetApplicationById(context.Background(), &applicationSecond)
+
+			Expect(err).To(BeNil())
+			Expect(applicationFirst.SeqID).To(Equal(applicationSecond.SeqID))
+			Expect(applicationFirst.Spec_field).NotTo(Equal(applicationSecond.Spec_field))
+
+			clusterUser := db.ClusterUser{User_name: string(workspace.UID)}
+			err = dbQueries.GetClusterUserByUsername(context.Background(), &clusterUser)
+			Expect(err).To(BeNil())
+
+			gitopsEngineInstance := db.GitopsEngineInstance{Gitopsengineinstance_id: applicationSecond.Engine_instance_inst_id}
+			err = dbQueries.GetGitopsEngineInstanceById(context.Background(), &gitopsEngineInstance)
+			Expect(err).To(BeNil())
+
+			managedEnvironment := db.ManagedEnvironment{Managedenvironment_id: applicationSecond.Managed_environment_id}
+			err = dbQueries.GetManagedEnvironmentById(ctx, &managedEnvironment)
+			Expect(err).To(BeNil())
+
+			//############################################################################
+
+			// ----------------------------------------------------------------------------
+			// Delete the GitOpsDepl and verify that the corresponding DB entries are removed
+			// ----------------------------------------------------------------------------
+
+			err = k8sClient.Delete(ctx, gitopsDepl)
+			Expect(err).To(BeNil())
+
+			_, _, _, _, err = appEventLoopRunnerActionSecond.applicationEventRunner_handleDeploymentModified(ctx, dbQueries)
```

Comment:
Yes, I can, though there is no harm if I it is not done, because in next step anyway I am checking that DB entries are deleted. At other places it is required because `applicationEventRunner_handleDeploymentModified`function is internally calling different handlers (i.e. `handleNewGitOpsDeplEvent`, `handleUpdatedGitOpsDeplEvent`), so there is no way to find out which handler was called and entries in respective tables don't hep either.